src/hooks/useWalletSupportsChain/useWalletSupportsChain.ts

```diff
@@ -51,7 +53,7 @@ export const walletSupportsChain: UseWalletSupportsChain = ({ chainId, wallet })
     case gnosisChainId:
       return supportsGnosis(wallet)
     case cosmosChainId:
-      return supportsCosmos(wallet)
+      return isSnapsEnabled || supportsCosmos(wallet)
```

Comment:
@pastaghost snaps hdwallet will need to `_supportsCosmos: true` to be able to remove this (same goes for other chains), but we have to make sure that this is programmatic and the base MM without flasks is still set to false